src/main/java/homework_3/library/LibraryCard.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * List of ImmutableClass requirements:
+ * 1) class must be final;
+ * 2) all fields must be final;
+ * 3) methods-mutators must return a new copy of the object;
```

Comment:
Hm... it's not true. Your class is easily mutable, btw

Reply:
Fixed (in setters "this.author" replaced with "this.author.clone()").